perfkitbenchmarker/pkb.py

```diff
@@ -491,4 +504,10 @@ def Main(argv=sys.argv):
         '%s\nUsage: %s ARGS\n%s', e, sys.argv[0], FLAGS)
     sys.exit(1)
 
+  try:
+    SetUpPKB()
+  except Exception as e:
+    logging.error('Got exception %s', e)
+    sys.exit(1)
```

Comment:
I did it this way to imitate the way the old code returned. However, you're completely right. Changed.